--- dist/macosx/include/OpenImageIO/imagecache.h	2016-01-28 18:48:01.000000000 -0800
+++ dist/macosx/include/OpenImageIO/imagecache-native.h	2016-02-28 01:01:33.000000000 -0800
@@ -184,11 +184,11 @@
     /// not be opened as an image file by any available ImageIO plugin.
     virtual bool get_imagespec (ustring filename, ImageSpec &spec,
                                 int subimage=0, int miplevel=0,
-                                bool native=false) = 0;
+                                bool nativefmt=false) = 0;
     virtual bool get_imagespec (ImageHandle *file, Perthread *thread_info,
                                 ImageSpec &spec,
                                 int subimage=0, int miplevel=0,
-                                bool native=false) = 0;
+                                bool nativefmt=false) = 0;
 
     /// Return a pointer to an ImageSpec associated with the named image
     /// (the first subimage & miplevel by default, or as set by
@@ -202,11 +202,11 @@
     /// as long as nobody (even other threads) calls invalidate() on the
     /// file, or invalidate_all(), or destroys the ImageCache.
     virtual const ImageSpec *imagespec (ustring filename, int subimage=0,
-                                        int miplevel=0, bool native=false) = 0;
+                                        int miplevel=0, bool nativefmt=false) = 0;
     virtual const ImageSpec *imagespec (ImageHandle *file,
                                         Perthread *thread_info,
                                         int subimage=0, int miplevel=0,
-                                        bool native=false) = 0;
+                                        bool nativefmt=false) = 0;
 
     /// Retrieve the rectangle of pixels spanning [xbegin..xend) X
     /// [ybegin..yend) X [zbegin..zend), with "exclusive end" a la STL,
