--- dist/macosx/include/OpenImageIO/imagebufalgo_util.h	2016-02-28 00:53:47.000000000 -0800
+++ dist/macosx/include/OpenImageIO/imagebufalgo_util-comment.h	2016-02-28 00:54:49.000000000 -0800
@@ -76,7 +76,7 @@
 ///     void my_image_op (ImageBuf &out, const ImageBuf &in,
 ///                       float scale, ROI roi);
 /// Then you can parallelize it as follows:
-///     ImageBuf R /*result*/, A /*input*/;
+///     ImageBuf R (result), A (input);
 ///     ROI roi = get_roi (R.spec());
 ///     parallel_image (bind(my_image_op,ref(R), cref(A),3.14,_1),
 ///                     roi);
